websub-native/src/main/java/io/ballerina/stdlib/websub/RequestHandler.java

```diff
@@ -0,0 +1,12 @@
+package io.ballerina.stdlib.websub;
+
+import io.ballerina.runtime.api.values.BObject;
+
+/**
+ * {@code RequestHandler} is a wrapper object used for service method execution.
+ */
+public class RequestHandler {
```

Comment:
Why do we need this ?

Reply:
* As per the discussion over the `websubhub` module same logic applies here.